force-app/main/default/classes/Trigger Recipes/MetadataTriggerHandler.cls

```diff
@@ -0,0 +1,155 @@
+ * allow you to declare the *order of methods* within the triggerHandler classes
+ * themselves. When using the MetadataTriggerHandler, it's better to have a
+ * high number of singularly focused trigger handler classes than a few classes
+ * with multiple methods.
+ *
+ */
+public inherited sharing class MetadataTriggerHandler extends TriggerHandler {
+    /**
+     * @description Overrides the standard Run() method, which allows this
+     * metadata based trigger handler can be an incremental update / sit beside
+     * other trigger handlers classes that are directly invoked by a trigger
+     */
+    @SuppressWarnings('PMD.ApexCRUDViolation')
+    override public void run() {
+        /**
+         * This is standard validation checking for triggers / triggerhandlers
+         */
+        if (!validateRun()) {
+            return;
+        }
+
+        addToLoopCount();
+
+        /**
+         * End standard validation
+         */
+
+        String objType = getSobjectType();
+        /**
+         * This query finds an ordered list trigger handler classes
+         * to execute. It ignores any classes that are marked as disabled.
+         *
+         * Note: It will exclude any triggerHandler metadata records for which
+         * the user's email address is found in a related disabled_for__mdt
+         * record.
+         *
+         * Admin's and Developers can selectively disable trigger handlers
+         * for all or selected individuals *without* deploying.
+         *
+         */
+        List<Metadata_Driven_Trigger__mdt> tryggers = [
+            SELECT Class__c
+            FROM Metadata_Driven_Trigger__mdt
+            WHERE
+                Object__c = :objType
+                AND Enabled__c = TRUE
+                AND Id NOT IN (
+                    SELECT Metadata_Driven_Trigger__c
+                    FROM Disabled_for__mdt
+                    /**
+                     * Note: The use raised on stream for this feature - to
+                     * selectively disable triggers per e-mail is different than
+                     * what's implemented here. Email addresses do not have to
+                     * be unique, but usernames are. You may want to use email
+                     * if, for instance, you want to disable all integration
+                     * users who share an email address.
+                     *
+                     * You can use any of the UserInfo.* methods to make this
+                     * kind of decision, so long as the CMT is able to represent
+                     * the data. You could use a UserID, but that's not portable
+                     * across environments/orgs and it's less likely, in my
+                     * estimation, that you'd typo an email address you can read
+                     * rather than an id that is seemingly random
+                     *
+                     */
+                    WHERE UserEmail__c = :UserInfo.getUsername()
```

Comment:
Shouldn't this field name be `User_Email__c` with an extra underscore in order to be consistent with other fields names?